index.d.ts

```diff
@@ -6,7 +6,7 @@ interface MakeEtaOptions {
   ignoreSameProgress?: boolean,
 }
 
-interface ETAInstance {
+export interface ETAInstance {
```

Comment:
unfortunately, this will produce `TS2309: An export assignment cannot be used in a module with other exported elements.` in the declaration file